D1673/P1673.md

```diff
@@ -6889,17 +6889,6 @@ or other things as appropriate.
 
 ### BLAS 1 functions [linalg.algs.blas1]
 
-<i>[Note:</i> The BLAS developed in three "levels": 1, 2, and 3.
```

Comment:
@crtrott @dalg24 We might want to consider

* changing the section titles not to refer to BLAS 1, 2, or 3; and/or
* moving this Note to the front matter, so that readers have some context for the section titles.

Either option suggests a more elaborate change than just removing the note.  Would you concur?